functional-test/src/test/groovy/org/rundeck/tests/functional/selenium/execution/CommandSpec.groovy

```diff
@@ -152,4 +154,25 @@ class CommandSpec extends SeleniumBase {
             executionShowPage.viewButtonNodes.isDisplayed()
     }
 
+    def "save as job button saves the command as a job"() {
+        when:
+        def commandPage = go CommandPage, SELENIUM_BASIC_PROJECT
+        def jobCreatePage = page JobCreatePage
+        def jobShowPage = page  JobShowPage
+        then:
+        commandPage.nodeFilterTextField.click()
+        commandPage.nodeFilterTextField.sendKeys".*"
+        commandPage.filterNodeButton.click()
+        commandPage.waitForElementToBeClickable commandPage.commandTextField
+        commandPage.commandTextField.click()
+        commandPage.waitForElementAttributeToChange commandPage.commandTextField, 'disabled', null
+        commandPage.commandTextField.sendKeys "echo running test '" + this.class.name.toString() + "'"
+        commandPage.runButton.click()
+        commandPage.saveAsJobButton.click()
+        jobCreatePage.getJobNameInput().sendKeys("From Command")
+        jobCreatePage.saveJob()
+        expect:
+        jobShowPage.validatePage()
```

Comment:
here we could add a validation for the job to actually run after being saved

Reply:
The scope of this test is to ensure that a command can become a job. It does that.
Execution of jobs is not in scope and is expected to be tested in the corresponding test surface.